dlk/python/dlk/templates/src/pack_input_to_qwords.cpp

```diff
@@ -60,7 +60,7 @@ void pack_input_to_qwords(QUANTIZED_NOT_PACKED input[],
     if (bit_count == nbit_qinput_word)
```

Comment:
As we trying to be generic and type-safe
How about making `nbit_qinput_word` to be a trait of `QuantizedPacked` class here
I.e.
```cpp
template <typename pack_type>
￼class QuantizedPacked {
...
  static constexpr BitCount = sizeof(pack_type) * 8;
```

```cpp
if (bit_count == QUANTIZED_PACKED::BitCount) {
```

So relation between number of bits in the type and the condition will be more clear

Reply:
Defined `BitCount`.